include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -100,6 +100,39 @@ def ExpandWhens : Pass<"firrtl-expand-whens", "firrtl::FModuleOp"> {
   let constructor = "circt::firrtl::createExpandWhensPass()";
 }
 
+def InferMemories : Pass<"firrtl-infer-memories", "firrtl::FModuleOp"> {
```

Comment:
Any suggestion for a better name?  In our previous notes we were calling it "LowerBehavioralMemories" which I like better, although we have been calling them CHIRRTL memories more often than behavioral memories.

Reply:
I might end up folding SFC's `InferReadWrite` pass functionality into this, so the name is probably ok.  `InferReadWrite` combines separate `Read` and `Write` ports into a single `ReadWrite` port)

Reply:
How about `lower-chirrtl` ?